Widgets/widget-MyTasks-old.json

```diff
@@ -0,0 +1,29 @@
+{
+  "id": "my-tasks",
```

Comment:
mistake #2?

Reply:
No, should have the same ID as the other one, since they are targeted for different versions